### ISSUE TITLE
command-not-found does not output anything if no command matches

### ISSUE THREAD

**Issue by bebehei** [metti/command-not-found#10]
Hi, the current command-not-found-handler has a small flaw. If command-not-found does not output anything, because it does not find similar commands, there is **no** error message. The only error-message is the exit-code (which most user do not actively poll).

Example in my case: `xfbuttons.sh` is in my own $PATH and it usually should exit without any message. Now for $some_reasons, if I mistype the command, `cnf-lookup` won't print out anything, because there is no matching command in the package-repos. And in this case, `command-not-found-handler` won't output anything.

```
[bebe:~] % xfbuttons.sh
[bebe:~] % xfbuttons.sh1
[bebe:~] 127 %
```

I actually did not find a good solution to easily patch `command-not-found-handler` in combination with `cnf-lookup`. Else I would have added a PR.

**Comment by metti** [comment]
That indeed is not the expected behaviour. Let me see what can be done.

**Comment by bebehei** [comment]
After thinking about this and breaking up my constraints ( 🤦‍ ) , I've seen the easy and correct fix. See #11

**Comment by metti** [comment]
fixed in 0.6.1

**Comment by bebehei** [comment]
Thanks for also tagging a new release!
